pkg/api/v1/register.go

```diff
@@ -39,7 +39,7 @@ func init() {
 
 // Adds the list of known types to api.Scheme.
 func addKnownTypes() {
-	api.Scheme.AddKnownTypes("v1",
+	api.Scheme.AddKnownTypes(api.Group, "v1",
```

Comment:
If we are going to separate register.go for different groups, i.e. if we have pkg/api/infra/v1/register.go, then here it will be 
```
api.Scheme.AddKnownTypes( "infra/v1",
```

I haven't read the newest discussion of this issue. Ignore me if this doesn't make sense.